.github/workflows/_buildpacks-release.yml

```diff
@@ -201,6 +201,9 @@ jobs:
       - name: Install Pack CLI
         uses: buildpacks/github-actions/setup-pack@v5.5.4
 
+      - name: Install Crane
+        uses: uses: imjasonh/setup-crane@v0.3
```

Comment:
We should be cautious about using Actions from third parties, particularly when they don't seem to be very popular repos, and the Action has access to our most sensitive credentials (such as the Docker Hub tokens).

For installing crane elsewhere in our automation, we use the `setup-tools` Action from upstream CNB:
https://github.com/heroku/languages-github-actions/blob/d3dcac71c07fcb38eee51d53a9b861d7abffe0e3/.github/workflows/_buildpacks-release.yml#L258-L259

You'll need to pass a specific `crane-version` if you want newer than the default, since the default version is a bit stale, xref:
https://github.com/buildpacks/github-actions/pull/262

Reply:
Yeah, I vetted this action. While it's not super popular, `imjasonh` is one of the primary devs on `crane` and regularly creates releases. By using `crane` at all, we're already exposed to this same user. The action is also version locked and the action source does exactly what you'd expect. 

I don't have any issue using the buildpacks one either, though. I'll switch it.